pkg/analyze/analyzer.go

```diff
@@ -441,6 +441,9 @@ func Analyze(analyzer *troubleshootv1beta2.Analyze, getFile getCollectedFileCont
 		if err != nil {
 			return nil, err
 		}
+		if result == nil {
```

Comment:
I'm guessing this needs to return a slice with a nil result else [this](https://github.com/replicatedhq/troubleshoot/blob/main/pkg/analyze/download.go#L42) part with panic

Like so

```go
		if result != nil {
			result.Strict = analyzer.CephStatus.Strict.BoolOrDefaultFalse()
		}
		return []*AnalyzeResult{result}, nil
```